module-2/backend/week_one.md

```diff
@@ -5,30 +5,105 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+
+* DELETE: delete , GET: render a view, POST: submit content, PUT: update content, PATCH: sililar to PUT, used in Rails.
 2. What is Sinatra?
+
+* Sinatra is a web-application framework. It allows the developer to configure things to his liking more than Rails.
 4. What is MVC?
+
+* Model, View, Controller. Model: holds our logic; Controller: serves as our router; View: displays content
+
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+
+* To be in sync with what Rails does? 
+
 6. What types of variables are accessible in our view templates without explicitly passing them?
+
+* instance
+
 7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
   
   ```ruby
   get '/horses' do
     erb :index
   end
+  
+  get '/names' do
+    erb :index, :locals => {:name => params[:Mr.Ed]}
+  end
+  
+  index.erb
+  <% @count %>
   ```
 
 8. In the same code block, how would I pass a local variable `name` with a value of `Mr. Ed`?
 9. What's the purpose of ERB?
+
+* Make our Ruby code readible by the browser.

```

Comment:
Small note: our ERB files get parsed server side, so that by the time we send them to the browser all it's seeing is HTML. I would say that the above is correct, but wanted to make sure we had the full picture.